_docs/development.md

```diff
@@ -69,6 +79,7 @@ pinned dependencies for RapidPro can be found in ```pip-freeze.txt```. You can
 build the needed environment as follows (from the root rapidpro directory):
 
 {% highlight bash %}
+$ sudo apt-get install python-virtualenv postgresql-server-dev-9.3 python-dev ncurses-dev
 $ virtualenv env
```

Comment:
This is installing into the system python libraries, which we probably don't want to recommend. (the one exception being virtualenv, the others are part of pip-freeze and shouldn't be required)